crates/bevy_ecs/src/schedule/graph_utils.rs

```diff
@@ -8,22 +8,37 @@ use fixedbitset::FixedBitSet;
 
 use crate::schedule::set::*;
 
+/// Unique identifier for a system set stored in a [`ScheduleGraph`].
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct SystemNodeId(pub usize);
```

Comment:
I'd probably prefer if these fields were locked down: just readers and a `new` constructor, not mutating access.